llvm/test/Transforms/StraightLineStrengthReduce/NVPTX/slsr-i8-gep.ll

```diff
@@ -0,0 +1,155 @@
+; RUN: opt < %s -passes=slsr -S | FileCheck %s
```

Comment:
Please use utils/update_llc_test_checks.py to generate test checks.

Reply:
Done, thanks!

Reply:
We have check autogenerator for opt, too: utils/update_test_checks.py

Looks like we even have utils/update_any_test_checks.py, but I'm not sure if it works with the tests that mix llc and opt tests.

I don't have a strong opinion whether autogenerated tests for opt are better than handwritten checks. I'd give it a try, and see if that's an improvement or not.

Reply:
The  update_llc_test_checks comment seems to prevent update_test_checks.py. Maybe we can keep the handwritten opt checks.